core/storage/job/gae_models.py

```diff
@@ -89,6 +89,12 @@ def get_new_id(cls, entity_name):
 
     @property
     def is_cancelable(self):
+        """Checks if the job is cancelable.
+
+        Returns:
+            bool. True if the status_code is queued or started, otherwise
```

Comment:
bool. Whether the status-code is 'queued' or 'started'.

Reply:
I've changed the description to this. Thanks!